src/Form/index.tsx

```diff
@@ -243,10 +248,17 @@ function Form({
   children,
   _draft = false
 }: InternalProps & Props) {
+  // TODO: remove support for formName
+  if (formNameProp)
```

Comment:
i think it's fine to leave te warning here even if there are re-renders but we can move to a side effect if need be

Reply:
i think there are a lot of re-renders so IMO a side effect would be a better experience